src/Mod/Draft/draftutils/gui_utils.py

```diff
@@ -121,20 +119,30 @@ def autogroup(obj):
                 elif active_arch_obj:
                     active_arch_obj.addObject(obj)
                 elif active_part:
+                elif FreeCADGui.ActiveDocument.ActiveView.getActiveObject("part", False) is not None:
                     # add object to active part and change it's placement so accordingly
-                    # so object does not jump to different position
-                    inverse_placement = active_part.getGlobalPlacement().inverse()
-                    if get_type(obj) == 'Point':
+                    # so object does not jump to different position, works with App::Link
+                    # if not scaled. Modified accordingly to realthunder suggestions
+                    p, parent, sub = FreeCADGui.ActiveDocument.ActiveView.getActiveObject("part", False)
+                    matrix = parent.getSubObject(sub, retType=4)
+                    if matrix.hasScale() == 1:
+                        FreeCAD.Console.PrintMessage(translate("Draft",
+                                                               "Unable to insert new object into"
```

Comment:
You have to add a space at the end of the string so that the string is concatenated with the space.
```
"Unable to insert new object into "
"a scaled part"
```